js-plugins/visualizer/src/node/dataSource.ts

```diff
@@ -21,9 +21,7 @@ export function createDateSourceMiddleware(compiler: Compiler) {
         res.end(JSON.stringify(result));
       };
 
-      if (pathname === '/__record/modules') {
```

Comment:
Is this an expected change? I see the `/__record/modules` interface is still used by some modules.